src/Common/src/Interop/Unix/System.Private.CoreLib.Native/Interop.MemAllocFree.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal unsafe partial class Sys
+    {
+        [DllImport(Interop.Libraries.CoreLibNative, EntryPoint = "CoreLibNative_MemAlloc")]
+        internal static extern IntPtr MemAlloc(UIntPtr sizeInBytes);
+                
+        [DllImport(Libraries.CoreLibNative, EntryPoint = "CoreLibNative_MemFree")]
```

Comment:
Nit: This line has `Libraries.CoreLibNative` vs. the previous line `Interop.Libraries.CoreLibNative`. It would be nice to be consistent, at least within one file.